### DISCUSSION TITLE
Freedom GPT

### DISCUSSION THREAD

**Discussion by LordRelentless** [lean-dojo/LeanDojoChatGPT#7] [Ideas]
Is there a way you can add this as an extension to the FreedomGPT standalone app for Windows?

**Comment by yangky11** [comment]
We don't plan to do that in the near future, but feel free to do it by yourself.

**Reply by LordRelentless** [reply]
I'd love to do it myself, but I have no idea how. The reason I ask is because the Freedom GPT which is based on ChatGPT or Bard AI (they use multiple models) can operate stand alone with GPU or CPU AI acceleration. So having your system as an extension would make a great deal of sense for Freedom GPT.
